docs/_articles/en/codebuilder/cb-diffs.md

```diff
@@ -0,0 +1,24 @@
+---
+title: What's Different
+lang: en
+---
+## What’s Different in Code Builder
+
+Code Builder offers the same rich tooling and unleashes the power of Salesforce VS Code extensions  as the desktop version. No matter whether you are using VS Code on desktop or Code Builder from a browser, the Salesforce extensions you access are the same. While the extensions generally function the same way regardless of how they are being accessed, there are some nuances to be aware of:
+
+* Code Builder installed as a managed package, and your Code Builder environment comes pre-configured for you.
+* Working in Code Builder doesn’t require you to install VS Code or Salesforce Extensions or Salesforce CLI. 
+* Unlike VS Code for the desktop, you cannot add any new extensions to your Code Builder development environment.
+* Because Code Builder is inherently lightweight and optimized to run in the cloud, access to certain functionalities is limited:    
+    * Functions development is available for Javascript and Typescript Functions only.
+    * LWC Local Development (beta) is not available in Code Builder. Deploy LWC components to your org to view them.
+* Code Builder allows you to carry your development environment with you. Bookmark your development environment to return to it, or access the link from a different machine. 
+* Unlike a desktop IDE, Code Builder saves your work for you without you explicitly having to do so. But since your beta environment's usage is capped at 20 hours, and your environment is wiped once your clock runs out, remember to either push your work to source control or deploy your changes to your org. 
```

Comment:
do you want to say "commit" here instead of push like you do in the Quick Start?